.github/workflows/ci.yml

```diff
@@ -19,6 +19,7 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
-          cache: "yarn"
+          cache: 'yarn'
       - run: yarn
       - run: yarn lint
+      - run: yarn format-check
```

Comment:
If I understand, this command is the check of the test files, right?

Reply:
For all files (including test files) actually.